### PR TITLE
Feat/local seo capabilities

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/web-ca`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-4903
| License          | BSD 3-Clause

## Description

- Use API to know if local SEO is available for the given a hosting
- Refactor to avoid multiple calls to the same API for the same hosting

### CODE REVIEW

File: packages/manager/apps/web/client/app/hosting/cdn/order/hosting-cdn-order.routing.js [hunk 1]
```diff
@@ -96,15 +97,9 @@ export default /* @ngInject */ ($stateProvider) => {
...
-        serviceName,
-        $translate,
-        HostingCdnOrderService,
-      ) =>
-        HostingCdnOrderService.getServiceOption(serviceName).catch((error) =>
-          goBackWithError(get(error, 'data.message', error)),
-        ),
+      serviceOption: /* @ngInject */ (availableOptions, goBackWithError) =>
+        find(availableOptions, { family: 'cdn' }) ||
+        goBackWithError('No serviceOption found'),
```
Comment (radireddy) [inline packages/manager/apps/web/client/app/hosting/cdn/order/hosting-cdn-order.routing.js:102]: no translation?

File: packages/manager/apps/web/client/app/hosting/hosting.routes.js [hunk 2]
```diff
@@ -12,6 +14,11 @@ export default /* @ngInject */ ($stateProvider) => {
       tab: null,
     },
     resolve: {
+      availableOptions: /* @ngInject */ (WucOrderCartService, serviceName) =>
+        WucOrderCartService.getProductServiceOptions(
+          'webHosting',
+          serviceName,
+        ).catch(() => []),
```
Comment (radireddy) [inline packages/manager/apps/web/client/app/hosting/hosting.routes.js:21]: you dont want to show error page on error?

Comment (marie-j) [reply packages/manager/apps/web/client/app/hosting/hosting.routes.js:21]: @radireddy In that case no, I don't think it is necessary to block the whole hosting dashboard if options cannot be fetched

File: packages/manager/apps/web/client/app/hosting/cdn/order/hosting-cdn-order.routing.js (hunk 1 above)
Comment (marie-j) [reply packages/manager/apps/web/client/app/hosting/cdn/order/hosting-cdn-order.routing.js:102]: I only moved the code around, I didn't change the behaviour

**Comment by marie-j** [comment]
Rebased on `feat/web-ca`

**Comment by marie-j** [comment]
Rebased on feat/web-ca

**Comment by marie-j** [comment]
Rebased on `feat/web-ca`

**Comment by marie-j** [comment]
Rebased on `develop` as the base branch changed

**Pull request merged**